docs/content/guides/dagster/parallel-computing-dask.mdx

```diff
@@ -0,0 +1,270 @@
+---
+title: Parallel computing with Dask on Dagster | Dagster Docs
+description: This guide illustrates how to parallel your compute with Dagster and Dask
+---
+
+# Parallel Computing with Dask on Dagster
+
+## What is Dask?
+
+[Dask](https://www.dask.org) is a flexible library for parallel computing in Python which can be a helpful tool when performing larger data engineering and machine learning tasks. This guide will review how to use [Dagster’s resources](https://docs.dagster.io/concepts/resources) to automate your Dask computations and make your pipelines faster. In this guide, you’ll learn how to:
+
+- Set up a Dask resources
+- Use Dask to speed up data assets
+- Use Dask’s machine learning tools to build a model
+
+---
+
+## Using Dagster’s resources to set up a Dask Cluster
+
+When using Dask with Dagster, you can use the `DaskResource` to set up a Dask client and simplify a pipeline that uses Dask across multiple assets. In this example, a local`dask.distributed` client is set up as a `ConfigurableResource`. Resources allow you to centralize your configuration in a single location, reducing redundant code in your assets.
```

Comment:
Add space between "local" and "dask.distributed" I think.

Reply:
> you can use the `DaskResource`

This makes it sound like Dagster provides its own `DaskResource`. I think something like "You can define a `DaskResource` to share Dask configuration across multiple assets" would be more accurate.